From 2459f3c646fb1ad679f5de1df08965a659b5ea77 Mon Sep 17 00:00:00 2001
From: julee <julee.vv@outlook.com>
Date: Thu, 24 Dec 2015 17:46:46 +0800
Subject: [PATCH] port to android

---
 ev_select.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ev_select.c b/ev_select.c
index f38d6ca..234d204 100644
--- a/ev_select.c
+++ b/ev_select.c
@@ -65,6 +65,10 @@
 # define NFDBYTES (NFDBITS / 8)
 #endif
 
+#ifdef ANDROID
+typedef long fd_mask;  //FIXME:
+#endif
+
 #include <string.h>
 
 static void
-- 
2.5.0

